Dockerfile

```diff
@@ -1,13 +1,22 @@
 FROM alpine
 MAINTAINER Rémi Alvergnat <toilal.dev@gmail.com>
 
-RUN apk add --update python py-pip ca-certificates && rm -rf /var/cache/apk/*
+RUN apk add --update python3 ca-certificates nodejs build-base python3-dev libffi-dev openssl-dev unzip && \
+    rm -rf /var/cache/apk/*
 
 ADD . /opt/flexget
 WORKDIR /opt/flexget
 
-RUN pip install paver
-RUN pip install -e .
+RUN pip3 install paver
```

Comment:
Is paver needed for something else? We haven't used it for flexget for quite a while.

Reply:
Paver is not needed for sure. Also you might want to chain these run commands together (&&) as each command creates unnecessary docker layer.

Reply:
Yes, just forgot to remove it. Done.